.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+### Jira Issue
```

Comment:
Can we move this down a bit or will it break the Jira integration? Although at this point most PRs will have an associated Jira, this is an open source repo and we should prepare for contributors outside of Trustly.

Reply:
Hi, @lukevance. Yeah, good point!

I can move it down, flag it as optional, or even "merge" it with the Additional Information topic.

Maybe something like:

_(Insert any additional information related to this pull request, as more context or Jira ticket. Leave it as **"None"** or **"Not applicable"** if you have nothing to add)_

I don't think that the integration will break, but I also don't have a previous experience with this.

Reply:
I like the idea of merging it, as long as it is obvious for our team where the JIRA should go but not confusing to external contributors, that is the goal 👍🏻

Reply:
Updates applied!